Jenkinsfile

```diff
@@ -15,6 +9,23 @@ pipeline {
                 }
             }
             stages {
+                stage('set_version') {
+                    steps {
+                        sh 'bumpversion patch'
+                    }
+                }
+                stage('release') {
+                    when { branch 'master' }
+                    steps {
+                        withCredentials([usernamePassword(credentialsId: env.CREDENTIALS_ID, passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')]) {
+                            sh '''
+                                export VERSION=$(bump2version --list --allow-dirty release | grep -oP '^new_version=\K.*$')
```

Comment:
As discussed, I think we should move all the version bumping scripts into a `bumpversion.sh` script in the repo for consistency and easier testing.